ia/src/ml_agent.py

```
@@ -204,22 +271,76 @@ def strategy(client):
             else:
-                if random.random() < 0.7:
+                if random.random() < 0.5:
                     protocole.execute_command(client, utils.RIGHT, None)
                 else:
                     protocole.execute_command(client, utils.FORWARD, None)
-
-    elif food_count < 20:
+        protocole.execute_command(client, utils.INVENTORY, None)
+        return
+    
+    if client["help_status"] == True:
```

Ruff (0.11.9)

Avoid equality comparisons to `True`; use `if client["help_status"]:` for truth checks

Replace with `client["help_status"]`

(E712)